### DISCUSSION TITLE
Consider wiki/ as the default folder instead of .github/wiki

### DISCUSSION THREAD

**Discussion by jcbhmr** [spenserblack/actions-wiki#33] [Ideas]
As stated in #31, the default is `.github/wiki` because it's specific to GitHub. That was a "why is it this way" post, this is my "rebuttal" or alternative proposal idea as to why I think `wiki` is better:

It sorta comes down to your semantic opinion. Is it a generic wiki, or is it _github specific_? Like for instance, would you have a GitLab ↔ GitHub mirror with _separate wikis_? If so, then `.github/wiki/**` makes perfect sense to parallel a `.gitlab/wiki/**` so that each gets a custom wiki deployment. If you'd use the same wiki, then something like a more generic `wiki/` that _just so happens_ to get deployed to GitHub wiki makes more sense in my brain 🧠

sorta like how a docs/ folder _can_ be deployed to github pages, but it's not in `.github/pages/**`. It's like a generic file folder that _just so happens_ to be deployed to Vercel, or Netlify, or in this fictitious scenario, github pages.

That and it improves discoverability if you clone the docs locally like in vscode since most people don't touch the .github folder when working on non-CI stuff

all in all, I think it just makes more semantic sense to have _the default_ be `wiki` vs `.github/wiki` since `wiki` is more generic.

But that's my opinion. 🤷‍♂️

**Comment by spenserblack** [comment]
One thing to consider is that `.github/`, and other hidden folders, is often used for files that developers usually don't need to reference, and users never need to reference (directly). For example, [assets for the README](https://github.com/search?q=path%3A%2F%5C.github%5C%2Fassets%2F&type=code). Since readers should reference the actual Wiki, and coders will probably make more contributions to the main project than the wiki.

Using GH Pages as an example, yes, you can deploy from `docs/` which is visible, but you can also deploy from an orphaned branch `gh-pages`, which is *very* hidden, and specific to GitHub :laughing:

:thinking: That gives me an idea for deploying from a `gh-wiki` branch...

**Reply by jcbhmr** [reply]
❤ i can concede that there is some legitimate precedent surrounding non-github-specific stuff in .github just to "stash it" out of the way (which is debatable whether or not that's good 🤷‍♂️, but it's a good point that it **does exist as a prior convention**)

a point in favor of wiki vs .github/wiki is its prevalence in existing repos and the convention of where wiki docs are put: [**137,000** in `/wiki/**.md`](https://github.com/search?q=path%3A%2Fwiki%2F**.md&type=code) vs [**154** in `/.github/wiki/**.md`](https://github.com/search?q=path%3A%2F.github%2Fwiki%2F**.md&type=code) -- pretty stark difference 🤪

another might be that a lot of the other wiki actions default to docs/ or wiki/ or even ./ for deployments. the most popular one defaults to wiki/ https://github.com/Andrew-Chen-Wang/github-wiki-action/blob/master/action.yml#L11

**Reply by jcbhmr** [reply]
if possible, itd be nice to have a yes, maybe, or dont-bring-it-up-again answer to resolve #57 which is in limbo

**Reply by spenserblack** [reply]
Really old discussion, but just thought of this after following discussions on GitHub's new alert syntax. GitHub has now extended their markdown syntax, in a way that is very specific to GitHub and doesn't seem to be compatible with any non-GitHub client right now:

```markdown
> [!WARNING]
> This will not render properly on another site.
```

> [!WARNING]
> This will not render properly on another site.

The fact that GitHub is now diverging from popular Markdown syntax and making GFM more unique solidifies my opinion that GitHub wiki files should live under `.github/` by default.
